scripts/buildtools/commands/prepack.js

```diff
@@ -1,9 +1,34 @@
 const path = require('path');
 const fs = require('fs');
+const crypto = require('crypto');
 const shell = require('shelljs');
 const chalk = require('chalk');
 
 const BUILD_DIR = path.resolve(__dirname, '..', '..', '..', 'dist');
+const JS_DIR = path.join(BUILD_DIR, 'dist/js');
+const CSS_DIR = path.join(BUILD_DIR, 'dist/css');
+
+function generateSris () {
+  const jsFiles = fs.readdirSync(JS_DIR);
+  const cssFiles = fs.readdirSync(CSS_DIR);
+  return [
+    ...jsFiles.map(file => path.join(JS_DIR, file)), 
+    ...cssFiles.map(file => path.join(CSS_DIR, file))
+  ]
+  .filter(path => !path.endsWith('.map'))
+  .reduce((acc, path) => {
+    // get filename (key)
+    const parts = path.split('/');
+    const filename = parts[parts.length - 1];
+    const content = fs.readFileSync(path, { encoding: 'utf8' });
+    // generate sri (value)
+    const hash = crypto.createHash('sha384').update(content, 'utf8');
```

Comment:
PR description shows example output with `sha256-`, could you update to avoid confusion? Any reason to use `sha384` instead?

Reply:
sha384 is used in monolith, but technically it should not matter, as browser can intepret the hash alg